_meetup/16-Jan-2016.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: meetup
-title: "Meetup #15 - 16 Jan 2016"
+title: "Meetup #15 - Coming 16 Jan 2016"
```

Comment:
maybe we should add a logic if the date is in the future, the name attaches UPCOMING automatically. That is much better than this.